azure/database/postgres/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "azurerm_postgresql_flexible_server" "main" {
```

Comment:
sadly, it seems terraform does **not** expose the `public_network_access` setting. Let's output [public_network_access_enabled](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/postgresql_flexible_server#public_network_access_enabled) and print that so we can check if this is (hopefully) disabled by default.